Gemfile.lock

```diff
@@ -1,3 +1,6 @@
+GEM
+  specs:
```

Comment:
I run `bundle install` while I was at it and it generated this.

I tried removing it and running `bundle install` again, and it reappeared. I also verified it's still there after upgrading to the latest version of Bundler, 2.2.21.